src/apps/deskflow-gui/dialogs/AboutDialog.cpp

```diff
@@ -89,5 +89,11 @@ AboutDialog::AboutDialog(QWidget *parent) : QDialog(parent)
 
 void AboutDialog::copyVersionText()
 {
-  QGuiApplication::clipboard()->setText(m_versionString);
+  QString infoString = QStringLiteral("Deskflow: %1(%2)\nusing Qt: %3\non: %4")
+                           .arg(kVersion, kVersionGitSha, qVersion(), QSysInfo::prettyProductName());
+#ifdef Q_OS_LINUX
+  infoString.append(QStringLiteral("\nSession: %1(%2)")
```

Comment:
Nit: Might look nicer if there was a space.

```suggestion
  infoString.append(QStringLiteral("\nSession: %1 (%2)")
```